projects/plugins/jetpack/modules/likes.php

```diff
@@ -95,70 +101,65 @@ function __construct() {
 		add_action( 'save_post', array( $this->settings, 'meta_box_save' ) );
 		add_action( 'edit_attachment', array( $this->settings, 'meta_box_save' ) );
 		add_action( 'sharing_global_options', array( $this->settings, 'admin_settings_init' ), 20 );
-		add_action( 'sharing_admin_update',   array( $this->settings, 'admin_settings_callback' ), 20 );
+		add_action( 'sharing_admin_update', array( $this->settings, 'admin_settings_callback' ), 20 );
 	}
 
 	/**
 	 * Set the social_notifications_like option to `on` when the Likes module is activated.
 	 *
 	 * @since 3.7.0
-	 *
-	 * @return null
 	 */
-	function set_social_notifications_like() {
+	public function set_social_notifications_like() {
 		update_option( 'social_notifications_like', 'on' );
 	}
 
 	/**
 	 * Delete the social_notifications_like option that was set to `on` on module activation.
 	 *
 	 * @since 3.7.0
-	 *
-	 * @return null
 	 */
-	function delete_social_notifications_like() {
+	public function delete_social_notifications_like() {
 		delete_option( 'social_notifications_like' );
 	}
 
-
 	/**
 	 * Overrides default configuration url
 	 *
 	 * @uses admin_url
 	 * @return string module settings URL
 	 */
-	function jetpack_likes_configuration_url() {
+	public function jetpack_likes_configuration_url() {
 		return admin_url( 'options-general.php?page=sharing#likes' );
 	}
 
 	/**
 	 * Loads Jetpack's CSS on the sharing page so we can use .jetpack-targetable
 	 */
-	function load_jp_css() {
-		// Do we really need `admin_styles`? With the new admin UI, it's breaking some bits.
-		// Jetpack::init()->admin_styles();
+	public function load_jp_css() {
+		/**
+		* Do we really need `admin_styles`? With the new admin UI, it's breaking some bits.
+		* Jetpack::init()->admin_styles();
+		*/
 	}
 
 	/**
 	 * Load scripts and styles for front end.
-	 * @return null
 	 */
-	function load_styles_register_scripts() {
+	public function load_styles_register_scripts() {
 		if ( $this->in_jetpack ) {
 			wp_enqueue_style( 'jetpack_likes', plugins_url( 'likes/style.css', __FILE__ ), array(), JETPACK__VERSION );
 			$this->register_scripts();
 		}
 	}
 
-
 	/**
-     * Stub for is_post_likeable, since some wpcom functions call this directly on the class
+	 * Stub for is_post_likeable, since some wpcom functions call this directly on the class
 	 * Are likes enabled for this post?
-     *
-     * @param int $post_id
-     * @return bool
+	 *
+	 * @param int $post_id - id of the post.
```

Comment:
FYI, that `-` is not required (and normally not used) in PHP-land. It's only the JS eslint that requires it.